pkg/asset/ignition/bootstrap/cvoignore.go

```diff
@@ -51,21 +52,34 @@ func (a *CVOIgnore) Generate(dependencies asset.Parents) error {
 	var files []*asset.File
 	files = append(files, operators.FileList...)
 	files = append(files, openshiftManifests.FileList...)
+
+	seen := make(map[string]string, len(files))
 	for _, file := range files {
 		u := &unstructured.Unstructured{}
 		if err := yaml.Unmarshal(file.Data, u); err != nil {
 			return errors.Wrapf(err, "could not unmarshal %q", file.Filename)
 		}
+		group := u.GetObjectKind().GroupVersionKind().Group
+		kind := u.GetKind()
+		namespace := u.GetNamespace()
+		name := u.GetName()
+
+		key := fmt.Sprintf("%s |! %s |! %s |! %s", group, kind, namespace, name)
```

Comment:
Out of curiosity, why the `|!` separator?

Reply:
I didn't spend much time poking around in allowed-characters for the four entries, and I wanted a delimiter that was unlikely to show up in any of them.

Reply:
it would perhaps be more robust to just use `%q%q%q%q`?